west.yml

```diff
@@ -83,7 +83,7 @@ manifest:
       revision: 00dd4fab1a00f2f6e995ef3f2e7c3814689f8885
       path: modules/fs/nffs
     - name: hal_nxp
-      revision: c6ce9b7e8ddb501ca6d54e927e1602cb5f7a7183
+      revision: 1d5f331869c32e23c3c7523881aa8f0da95f0792
```

Comment:
wait that's a different change: you are bumping nxp's HAL to a newer verison. make a dedicated PR for it please.

Reply:
@tbursztyka without this version eth_mcux won't work, the version mentioned here has changes related to this patch.

Reply:
So make a version bump PR first, and then the eth_mcux PR dependent over it.

Reply:
@tbursztyka wouldn't a separate commit work rather than a whole new PR, there is no major SDK update

Reply:
Other option is to add a separate commit into this PR that updates the HAL version.

Reply:
the pb with the hal version bump is that it might affect other drivers that are not related to ethernet. Thus why it's imo preferable to separate completely

Reply:
But if we find out that the HAL version does not work everywhere we would be forced to revert both commits anyway so the end result is the same (as if we submit two separate PRs).

Reply:
@jukkar should be a good thing, one will not work without the other.

Reply:
https://docs.zephyrproject.org/latest/guides/modules.html#changes-to-existing-modules

You can use `revision: pull/24/head` to run CI testing. Once it passes, the zephyrproject-rtos/hal_nxp#24 can be merged and the revision here can be updated to the commit hash in the master (i.e. the commit that has been rebased onto master).